.travis.yml

```diff
@@ -34,7 +34,7 @@ addons:
 
 before_install:
   - rm -f Gemfile.lock
-  - gem install -v '~> 1.17' bundler
+  - gem install -v '~> 1.17, <2' bundler
```

Comment:
The change is unnecessary, the `~>`  would be keeping it in the `1.17.z` version, wouldn't it?

Reply:
I was having issues with bundler 2.0 getting installed.  Looking at the documentation [here](https://docs.travis-ci.com/user/languages/ruby/#bundler-20), maybe I should add an uninstall in case 2 is installed.